Fabric.Terminology.Domain/Services/ValueSetComparisonService.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Fabric.Terminology.Domain.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Fabric.Terminology.Domain.Models;
+
+    public class ValueSetComparisonService : IValueSetComparisonService
+    {
+        private readonly IValueSetService valueSetService;
+
+        public ValueSetComparisonService(
+            IValueSetService valueSetService)
+        {
+            this.valueSetService = valueSetService;
+        }
+
+        public Task<ValueSetDiffComparisonResult> CompareValueSetCodes(IEnumerable<Guid> valueSetGuids)
+        {
+            return this.CompareValueSetCodes(valueSetGuids, new List<Guid>());
+        }
+
+        public async Task<ValueSetDiffComparisonResult> CompareValueSetCodes(
+            IEnumerable<Guid> valueSetGuids,
+            IEnumerable<Guid> codeSystemGuids)
+        {
+            var valueSetGuidHash = valueSetGuids.ToHashSet();
+            var codeSysetmGuidHash = codeSystemGuids.ToHashSet();
```

Comment:
Spelling.
Did you know Visual Studio has a spell checker extension?

Reply:
I'll look for that now - never thought to look ;-)